FakeServer.Test/SortHelperTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using FakeServer.Common;
+using FakeServer.WebSockets;
+using Newtonsoft.Json;
+using System;
+using System.Dynamic;
+using System.Linq;
+using System.Threading;
+using Xunit;
+
+namespace FakeServer.Test
+{
+    public class SortHelperTests
+    {
+        [Fact]
```

Comment:
You can add test for a case when sortFields has an empty string

```cs
[Fact]
        public void SortFields_EmptySortArray()
        {
            var result = SortHelper.SortFields(new dynamic[] { }, new string[] { });
            Assert.NotNull(result);

            result = SortHelper.SortFields(new dynamic[] { }, new[] { "" });
            Assert.NotNull(result);
        }
```

Reply:
Also remove unnecessary usings statements and empty lines. I use http://www.codemaid.net/ for cleaning.

Reply:
Do you know of any good similar extensions for VS Code?

Reply:
Thanks very much for the feedback.  I think I have completed all of the changes you've requested other than the removing the unnecessary using statements and empty lines.  I'm currently using VS Code so CodeMaid isn't supported.  I'll continue to look for a good extension to complete that.